docs/tutorial.rst

```diff
@@ -90,6 +90,7 @@ You can create those fragments either by hand, or using the ``towncrier create``
 Let's create some example news fragments to demonstrate::
 
    $ echo 'Fixed a thing!' > src/myproject/newsfragments/1234.bugfix
+   $ towncrier create --content 'Fixed thing again!' 1234.bugfix  # since the file existed, this will create a 1234.bugfix.1
```

Comment:
I think that is best to describe the "counter" functionality in a separate paragraph/section.

This example is already quite heavy and is hard to focus on the core functionality vs extra features.

Thanks